source/viz.md

```diff
@@ -1169,41 +1262,36 @@ Bar plot of size for Earth's largest 12 landmasses.
 :::
 
 
-The plot in {numref}`islands_bar_top` is definitely clearer now,
-and allows us to answer our question
-("Which are the top 7 largest landmasses continents?") in the affirmative.
-But the question could be made clearer from the plot
-by organizing the bars not by alphabetical order
-but by size, and to color them based on whether they are a continent.
-The data for this is stored in the `landmass_type` column.
-To use this to color the bars,
-we use the `color` argument to color the bars according to the `landmass_type`
+The plot in {numref}`islands_bar_top` is clearer now,
+and allows us to answer our question:
+"Which are the top 7 largest landmasses continents?".
+However, we could still improve this visualization
+by organizing the bars by landmass size rather than by alphabetical order.
+We could also color the bars based on whether they are a continent or not
```

Comment:
remove "or not"

Reply:
"they are" -> "they correspond to"

Reply:
here is what i'm changing the R version to read:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/5981307/359570a7-7bf6-48a4-abc2-cdbb30185833)

Reply:
Updated based on your screenshot with a slight tweak in the wording to match altair's syntax